youtube.html

```diff
@@ -0,0 +1,430 @@
+    />
+    <link rel="stylesheet" href="yt.css" />
+  </head>
+
+  <body>
+    <header class="row header-grid bg-white">
+      <i class="fas fa-bars pl-1 toggler"></i>
+      <div class="brand">
+        <img class="brand-img" alt="YouTube" src="assets/logo.png" />
+        <sup class="super-script">MA</sup>
+      </div>
+      <form class="form-inline">
+        <input type="text" placeholder="Search" class="input-inline" />
+        <button type="submit" class="btn-inline ml--2">
+          <i class="fas fa-search" style="color: black;"></i>
+        </button>
+      </form>
+      <ul class="nav-list">
+        <li class="list-item">
+          <i class="fas fa-video mild"></i>
+        </li>
+        <li class="list-item">
+          <i class="fas fa-th mild"></i>
+        </li>
+        <li class="list-item">
+          <i class="fas fa-bell mild"></i>
+        </li>
+        <li class="list-item">
+          <button class="btn-blue white">Sign in</button>
+        </li>
+      </ul>
+    </header>
+
+    <div class="row banner my-1" id="lang-pref">
+      <div class="banner-header px-1 bg-white">
+        <p>
+          <i class="fas fa-user-edit"></i>
+          Choose your language.
+        </p>
+        <p>
+          <i class="fas fa-times"></i>
+        </p>
+      </div>
+      <div class="banner-body px-2 py-1 bg-white">
+        <p class="text-medium">
+          You're viewing YouTube in <b>English (US)</b>. You can
+          <a href="#" class="text-small">change this preference below.</a>
+        </p>
+        <p class="text-small"><a href="#" class="right">Learn more</a></p>
+      </div>
+    </div>
+
+    <main class="half-1 left-side p-1">
+      <div class="video-player">
+        <iframe
+          src="https://www.youtube.com/embed/C0oh-o1pY7A"
+          frameborder="0"
+          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </div>
+      <section class="caption bg-white p-1 mt-half">
+        <h2 class="caption-text">
+          10 Best Places to Visit in Morocco - Travel Video
+        </h2>
+        <div class="channel-body">
+          <img alt="maroc" class="channel-img pb-1" src="assets/maroc.jpg" />
+          =======
+          <img alt="maroc" class="channel-img pb-1" src="assets/maroc.jpg" />
+          >>>>>>> master
+          <div class="user-account pb-1">
+            <p class="channel-name text-medium">Morocco the land of coltur</p>
+            <div class="acc-subs">
+              <button type="button" class="btn subscribe m-0 white">
+                <img
+                  class="bg-youtube"
+                  src="assets/youtube-logo.png"
+                  alt="YouTube"
+                />
+                <img
+                  class="bg-youtube"
+                  src="assets/youtube-logo.png"
+                  alt="YouTube"
+                />
```

Comment:
remove one image button and leave only one

Reply:
thanks and done